source/GameInterface/Services/SiegeEngines/Handlers/SiegeEnginesContainerLifetimeHandler.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Common.Util;
+using GameInterface.Services.ObjectManager;
+using GameInterface.Services.SiegeEnginesContainers.Messages;
+using HarmonyLib;
+using Serilog;
+using static GameInterface.Services.BesiegerCamps.Extensions.BesiegerCampExtensions;
+using static TaleWorlds.CampaignSystem.Siege.SiegeEvent;
+
+namespace GameInterface.Services.SiegeEnginesContainers.Handlers;
+
+internal class SiegeEnginesContainerLifetimeHandler : IHandler
```

Comment:
Can you assign the appropriate issues to this MR from #1005? (One of the sub issues from that)

Reply:
Linked 1001 and 1003